tests/Feature/SettingsTest.php

```diff
@@ -27,11 +27,13 @@ public function users_can_update_their_profile()
                 'name' => 'Freek Murze',
                 'email' => 'freek@example.com',
                 'username' => 'freekmurze',
+                'twitter_handle' => 'freek-murze',
```

Comment:
Can a Twitter handle have a "-" character? I know you're trying to differentiate here with the username so maybe try "freektwitter" of something?